protocol/avdl/keybase1/chat_local.avdl

```diff
@@ -87,9 +88,24 @@ protocol chatLocal {
   chat1.InboxView getInboxLocal(union { null, chat1.Pagination } pagination);
   ThreadView getThreadLocal(chat1.ConversationID conversationID, boolean markAsRead, union { null, chat1.Pagination } pagination);
   void postLocal(chat1.ConversationID conversationID, MessagePlaintext messagePlaintext);
-  chat1.ConversationID newConversationLocal(chat1.ConversationIDTriple conversationTriple);
 
-  array<chat1.ConversationID> resolveConversationLocal(ConversationInfoLocal conversation);
+  // resolveConversationLocal resolves information related to a conversation.
+  // If conversation.id is given (non-zero), it is used to locate the
+  // conversation. Otherwise, tlfName, topicName, and topicType, if any of
+  // which are non zero value, are used to search for conversation. Any
+  // matching conversation is added to a list and returned to the caller.
+  array<ConversationInfoLocal> resolveConversationLocal(ConversationInfoLocal conversation);
```

Comment:
The `ConversationInfoLocal`s returned includes the `ConversationID`, which tells them apart (is this what you mena?), but the UI would still need to present to user both and let user choose one.